NOISSUE - Add `Expire` to running containers via dockertest

sprql opened the pull request:
If a test failed by a panic, purging a container would fail too.
To prevent it, we can use `Expire()` for a container. In this case, docker daemon will auto-remove it after a specified time.

drasko commented on authn/postgres/setup_test.go:
```diff
@@ -37,6 +37,12 @@ func TestMain(m *testing.M) {
 
+	// force remove the container in 60 seconds
+	err = container.Expire(60)
+	if err != nil {
```
Embed `err` definition in `if`:

```go
if err := container.Expire(expInterval); err != nil {
...
}
```

drasko commented on authn/postgres/setup_test.go:
```diff
@@ -37,6 +37,12 @@ func TestMain(m *testing.M) {
 	}
 
+	// force remove the container in 60 seconds
+	err = container.Expire(60)
```
Consts should be declared on the top of the file.

sprql committed:
NOISSUE - Add `Expire` to running containers via dockertest

If a test failed by a panic, purging a container would fail too.
To prevent it, we can use `Expire()` for a container. In this case, docker daemon will auto-remove it after a specified time.

Signed-off-by: Alexander Obukhov <dev@sprql.space>

sprql committed:
NOISSUE - remove unused constants

Signed-off-by: Alexander Obukhov <dev@sprql.space>

manuio reviewed (approved):
LGTM

drasko commented on authn/postgres/setup_test.go:
```diff
@@ -18,6 +18,7 @@ import (
 )
 
 const wrong string = "wrong-value"
+const expInterval = 60 // in seconds
```
Just a curiosity - how did we get to this interval and is it enough for test to be run in all cases?

sprql replied:
This is a good question because it depends on each case, on the other hand, you can set the value with a large margin. I just had to start with a certain value and it seemed to me that a minute would be fine. Maybe you have a suggestion?

nmarcetic committed:
Merge branch 'master' into feature/dockertest

dborovcanin reviewed (commented):
If the code panics (tests included), I would say that having dangling containers are the least problem. I also agree that setting this expiration to something like 1h or 2h will be safe in almost all the cases. What concerns me is that this can cause very nasty bugs that are hard to discover. If there is there a way to notify the user if the container is expired, I vote for accepting this PR with an increased expiration period. Otherwise, I would avoid this change for the aforementioned reasons.

Also, the warning at the beginning of the test executions would be fine.
